Validation/SiOuterTrackerV/src/OuterTrackerMCHarvester.cc

```diff
@@ -0,0 +1,623 @@
+#include "Validation/SiOuterTrackerV/interface/OuterTrackerMCHarvester.h"
+
+OuterTrackerMCHarvester::OuterTrackerMCHarvester(const edm::ParameterSet& iConfig)
+{
+}
+
+OuterTrackerMCHarvester::~OuterTrackerMCHarvester()
+{
+}
+
+// ------------ method called once each job just after ending the event loop  ------------
+void OuterTrackerMCHarvester::dqmEndJob(DQMStore::IBooker & ibooker, DQMStore::IGetter & igetter) {
+	using namespace edm;
+
+	// Global variables
+	TF1* fit = new TF1("fit", "gaus", -0.01,0.01);
```

Comment:
these may be memory leaks

Reply:
Thank you, @kpedro88! I will address these